src/test/java/uk/gov/companieshouse/insolvency/data/service/InsolvencyServiceImplTest.java

```diff
@@ -95,6 +95,26 @@ void when_invalid_insolvency_number_is_given_then_throw_exception() {
         Mockito.verify(repository, Mockito.times(1)).findById(Mockito.any());
     }
 
+    @Test
```

Comment:
Might make sense to add @DisplayName, but its up to you 👍

Reply:
In this file we've opted for the more exhaustive method naming approach so I've kept it the same. I do like the display name and use it when it's consistent with the test style and methods are not aptly named.